src/components/PageLevelBanner/PageLevelBanner.stories.tsx

```diff
@@ -1,111 +1,94 @@
-import type { StoryObj } from '@storybook/react';
+import type { StoryObj, Meta } from '@storybook/react';
 import React from 'react';
 
-import { PageLeveBanner, Variant } from './PageLeveBanner';
+import { PageLevelBanner } from './PageLevelBanner';
 import Button from '../Button';
 
 export default {
-  title: 'Molecules/Messaging/PageLeveBanner',
-  component: PageLeveBanner,
+  title: 'Molecules/Messaging/PageLevelBanner',
+  component: PageLevelBanner,
   args: {
     title:
       'New curriculum updates are available for one or more of your courses.',
+    description: (
+      <>
+        Summit Learning has a full-time team dedicated to constantly improving
+        our curriculum. To see the updates,{' '}
+        <Button
+          onClick={(event: any) => event.preventDefault()}
+          status="neutral"
+          variant="link"
+        >
+          click into the course
+        </Button>
+        .
+      </>
+    ),
   },
-};
-
-type Args = React.ComponentProps<typeof PageLeveBanner>;
+  render: ({ variant, ...other }) => {
+    return <PageLevelBanner variant={variant} {...other} />;
```

Comment:
Since `getDescription` is no longer being used, I don't think we need this render method anymore